morphia/src/test/java/org/mongodb/morphia/TestQuery.java

```diff
@@ -369,37 +391,67 @@ public void testElemMatchQueryCanBeNegated() {
 
         Mapper mapper = getMorphia().getMapper();
 
-        List<Key<PhotoWithKeywords>> keys = getDs().find(PhotoWithKeywords.class)
+        validate(pwk1, pwk2, pwk3, pwk4, mapper, getDs().find(PhotoWithKeywords.class)
```

Comment:
those are literally just refactorings of existing tests in to methods so i can use both elemMatch queries with the same asserts.  I didn't change what they were testing